MAUI/DataGrid/grid-events.md

```diff
@@ -131,6 +184,19 @@ private void dataGrid_CellEntered(object sender, DataGridCellEnteredEventArgs e)
 {% endhighlight %}
 {% endtabs %}
 
+## CellEntered Command and its Parameter
+When the cell is entered into the grid, the `CellEnteredCommand` triggered that binds to the command which you defined in your ViewModel. With this we can also use the `CellEnteredCommandParameter` to be passed to the `Command` when it is executed.
```

Comment:
The command will be invoked when the mouse pointer enters a cell in the SfDataGrid. The instance of `DataGridCellEnteredEventArgs` will be passed as the default value of the command's parameter. It can be customized by setting the `CellEnteredCommandParameter` property.